model/src/main/proto/question.proto

```diff
@@ -81,3 +81,17 @@ message AnsweredQuestionOutcome {
   // Whether the answer the learner submitted is the correct answer.
   bool is_correct_answer = 2;
 }
+
+// The final score and mastery calculations of the user after completing a training session.
+message QuestionAssessmentPerformance {
+  // User's total session score as a percentage.
+  double total_score = 1;
+
+  // User's score per skill for the training session.
+  // This maps a skill id to a score.
```

Comment:
Here & below: comment lines should be together rather than split up. Consider these to be part of the same sentence/paragraph that describes the one thing below it.